pkg/baseline/packageIndex.json

```diff
@@ -16,12 +16,14 @@
       "StableVersions": [
         "4.0.0",
         "4.1.0",
-        "4.3.0"
+        "4.3.0",
```

Comment:
@ericstj Not sure if I needed to keep 4.3.0 or simply override it with 4.3.1.

Reply:
Never delete versions from the index.  It's meant to be a private copy of package history and mirror nuget.org releases.

Reply:
+1

Reply:
Ok, thanks. I'll amend this PR.